test/outdated/goldens/newer_versions.txt

```diff
@@ -126,6 +134,8 @@ foo           *1.2.3   *1.3.0      3.0.0       3.0.0
 transitive dependencies
 transitive    *1.2.3   2.0.0       2.0.0       2.0.0   
 
+transitive dev_dependencies
+
```

Comment:
empty section, but it doesn't say `all up to date`

Reply:
Also if we have  `--no-dev-dependencies` let's not show this section.

Reply:
Hmm, both good points!

Reply:
Done, Done